src/dmd/astbase.d

```diff
@@ -3986,8 +3986,8 @@ struct ASTBase
         TRUST trust;                // level of trust
         Expressions* fargs;         // function arguments
         PURE purity = PURE.impure;
-        byte inuse;
         ushort funcFlags;
+        byte inuse;
```

Comment:
a better solution is to put this in the bitset

Reply:
unfortunately this is one of the rare `inuse` variable that can have 2 has value.